ee/app/license/server/license.ts

```diff
@@ -228,6 +229,14 @@ export function onLicense(feature: string, cb: (...args: any[]) => void): void {
 	EnterpriseLicenses.once(`valid:${ feature }`, cb);
 }
 
+export function onValidateLicenses(cb: (...args: any[]) => void): void {
+	if (isEnterprise()) {
+		return cb();
```

Comment:
You should not return here otherwise if this executes when enterprise is loaded and for some reason, it becomes invalid it will not re-execute the callbacks.